process/testing/winfv/setup-fv-capz.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+# This script sets up a k8s cluster with one Linux node and one Windows node and run windows FV test.
+# Usage - OS Felix fv, following variables need to be set
+#         AZURE_SUBSCRIPTION_ID
+#         AZURE_TENANT_ID
+#         AZURE_CLIENT_ID
+#         AZURE_CLIENT_SECRET
+#
+
+GIT_VERSION=$(git describe --tags --dirty --long --always --abbrev=12)
+CALICO_HOME=$(cd ../../../; pwd)
+CAPZ_LOCATION=$CALICO_HOME/process/testing/winfv/capz
+KUBECONFIG=$CALICO_HOME/process/testing/winfv/capz/kubeconfig
+export WIN_NODE_COUNT=1
+
+function prepare_crc(){
+# Clean up calico-ready-clusters if exists
+  if [ -d capz ]; then
+    rm -Rf capz;
+    echo "Removed calico-ready-cluster"
+  fi
+
+# Checkout calico-ready-clusters cluster-api/capz
+  git clone https://github.com/tigera/calico-ready-clusters.git
+  cp -r ./calico-ready-clusters/cluster-api/capz ./capz
+  rm -rf ./calico-ready-clusters
+  . $CALICO_HOME/process/testing/winfv/capz/export-env.sh
+}
+
+function start_cluster(){
+  make -C $CAPZ_LOCATION create-cluster
+  make -C $CAPZ_LOCATION install-calico RELEASE_STREAM=master HASH_RELEASE=true PRODUCT=calico
+  #Get Windows node ip
+  WIN_NODE=`kubectl get nodes -o wide -l kubernetes.io/os=windows --no-headers --kubeconfig $KUBECONFIG | awk -v OFS='\t\t' '{print $6}'`
+  export WIN_NODE_IP=${WIN_NODE: -1}
+  LINUX_NODE=`kubectl get nodes -l kubernetes.io/os=linux,'!node-role.kubernetes.io/control-plane' -o wide --no-headers --kubeconfig $KUBECONFIG | awk -v OFS='\t\t' '{print $6}'`
+}
+
+function upload_calico_node(){
+  $CAPZ_LOCATION/scp-to-node.sh $WIN_NODE_IP $KUBECONFIG c:\\k\\config
+  $CAPZ_LOCATION/scp-to-node.sh $WIN_NODE_IP $CALICO_HOME/node/dist/windows/$CALICO_NODE_IMAGE c:\\calico-node-windows.tar
+  #Imports calico-node-windows image from locally build image
+  $CAPZ_LOCATION/ssh-node.sh $WIN_NODE_IP 'ctr --namespace k8s.io images import --base-name calico/node-windows c:\calico-node-windows.tar --all-platforms' > output
+}
+
+function upload_fv(){
+  $CAPZ_LOCATION/scp-to-node.sh $WIN_NODE_IP $CALICO_HOME/process/testing/winfv/run-fv.ps1 c:\\run-fv.ps1
+  $CAPZ_LOCATION/scp-to-node.sh $WIN_NODE_IP $CALICO_HOME/felix/fv/win-fv.exe c:\\k\\win-fv.exe
+}
+
+function prepare_windows_image(){
+  make -C $CALICO_HOME/node image-windows WINDOWS_IMAGE=node-windows
+
+  if [[ $WINDOWS_SERVER_VERSION = "windows-2022" ]]; then
+    CALICO_NODE_IMAGE="node-windows-$GIT_VERSION-ltsc2022.tar"
+  else
+    CALICO_NODE_IMAGE="node-windows-$GIT_VERSION-1809.tar"
+  fi
+}
```

Comment:
I think we need to do the same with the `install-cni` initContainer image (i.e. create tars and patch the ds to use those), so that we're testing with any and all changes...